### ISSUE TITLE
Are HingeL1, HingeL2, SigmoidCrossEntropy and Infogain loss layers implemented?

### ISSUE THREAD

**Issue by sunformoon** [PrincetonVision/marvin#48]
Hi, I just downloaded the master branch of marvin.
I checked the code and found that there are no implementations for the loss layers mentioned above. Did I miss something?

Thank you!

**Comment by rspezialetti** [comment]
Hi, i have the same problem. Are they implemented?

**Comment by sunformoon** [comment]
@rspezialetti Last year, I switched to pytorch for the 3d conv purpose. And tensorflow should also support the 3d conv very well.
